packages/console/src/consts/cloud.ts

```diff
@@ -0,0 +1 @@
+export const isCloud = process.env.IS_CLOUD === 'true';
```

Comment:
Will refactor this globally with backend logic on https://github.com/silverhand-io/essentials/pull/38 merged.